core/src/main/java/org/apache/calcite/util/DateTimeStringUtils.java

```diff
@@ -25,6 +29,15 @@
 
   private DateTimeStringUtils() {}
 
+  /** The SimpleDateFormat string for ISO timestamps, "yyyy-MM-dd'T'HH:mm:ss'Z'"*/
+  public static final String TIMESTAMP_FORMAT_STRING1 =
+      "yyyy-MM-dd'T'HH:mm:ss'Z'";
+
+  /** The SimpleDateFormat string for ISO timestamps with precisions, "yyyy-MM-dd'T'HH:mm:ss
+   * .SSS'Z'"*/
+  public static final String TIMESTAMP_FORMAT_STRING2 =
+      "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'";
```

Comment:
Can we use more readable name for these formats ?

Reply:
How about ISO_DATETIME_SECONDS_FORMAT_STRING and ISO_DATETIME_MILLISECONDS_FORMAT_STRING?

Reply:
How about ISO_DATETIME_FORMAT and ISO_DATETIME_DECIMAL_FRACTION_FORMAT?

Reply:
`ISO_DATETIME_FORMAT` and `ISO_DATETIME_FRACTIONAL_SECOND_FORMAT`

Reply:
The format name have been updated. Thanks a lot for your time on this PR.